src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -51,13 +51,17 @@ private CallbackValidations() {
 
     @NotNull
     static Validation<String, String> hasCaseReference(CaseDetails theCase) {
-        Object caseReference = null;
+        Object caseObj = null;
+        String caseReference;
         return theCase != null
             && theCase.getData() != null
-            && (caseReference = theCase.getData().get(ATTACH_TO_CASE_REFERENCE)) != null
-            && (caseReference instanceof String)
-            ? valid((String) caseReference)
-            : internalError("no case reference found: %s", String.valueOf(caseReference));
+            && (caseObj = theCase.getData().get(ATTACH_TO_CASE_REFERENCE)) != null
+            && (caseObj instanceof String)
+            && (!(caseReference = ((String) caseObj).replaceAll("[^0-9]", "")).isEmpty())
```

Comment:
Is it ok that we accept a mix of digits and other characters, while it should always be just digits?

Reply:
well from the testing we can enter what is from the web page ..
I imagine that the user will just copy and paste from the web page the reference number
i.e.
`#102932-129831-198171  `
So this makes it up to us to make it into a parseable long